app/alarm/model/src/main/resources/alarm_preferences.properties

```diff
@@ -3,7 +3,7 @@
 # --------------------------------------
 
 # Kafka Server host:port
-server=localhost:9092
+server=localhost:19092
```

Comment:
Is this just from a local test run and should be back to the default of 9092?

Reply:
Yes, I will fix this... thanks

Reply:
reverted all ports to the standard defaults in the next commit.